core/src/main/java/com/datastax/oss/driver/api/core/metadata/Node.java

```diff
@@ -108,6 +109,11 @@
   @NonNull
   NodeState getState();
 
+  /**
+   * The last time that this node transitioned to the UP state, or -1 if it's not up at the moment.
+   */
+  long getUpSince();
```

Comment:
It isn't obvious that this is milliseconds instead of nanos and uses `System.currentTimeMillis()`.  Would it be worth renaming this `getUpSinceMillis()` or something to that extent?

Reply:
Absolutely